ext/ChibiOS_21.11.3/ext/lwip/src/include/lwip/errno.h

```diff
@@ -181,7 +181,7 @@ extern int errno;
 
 /* Define LWIP_ERRNO_STDINCLUDE if you want to include <errno.h> here */
 #ifdef LWIP_ERRNO_STDINCLUDE
-#include <errno.h>
+#include <sys/errno.h>

```

Comment:
This looks strange and generally I wouldn't modify the included projects... but I don't want to spend time trying to figure out a better solution. I see you already tried.